src/pygeoprocessing/routing/routing.pyx

```diff
@@ -3354,16 +3355,17 @@ def extract_strahler_streams_d8(
                 is_drain = 0
                 x_l = xoff + i
                 y_l = yoff + j
-                local_flow_accum = <long>flow_accum_managed_raster.get(
+                local_flow_accum = <double>flow_accum_managed_raster.get(

```

Comment:
Strictly speaking the `<double>` probably isn't needed since `.get()` already returns a double.  But it also isn't hurting anything by leaving it, and the compiler might well optimize it out anyways!